### DISCUSSION TITLE
Error running CI/CD with semantic release today

### DISCUSSION THREAD

**Discussion by milaabl** [semantic-release/semantic-release#2966] [Configuration]

**Comment by milaabl** [comment]
Please help me figure out the issue. It worked properly yesterday. 🙏

**Comment by travi** [comment]
you've given us no information about the failure you are seeing or any versions that you are actually using. because of that, there is very little we can do to help. the best i can provide is the suggestion to review the release notes of our recent releases in case you are targeting the latest version
